### PR TITLE
Amazon jp

### PR DESCRIPTION
**Pull request by delwinbest**
Adding CONF for Amazon.co.jp

### CODE REVIEW

File: config/fairgame.conf [hunk 1]
```diff
@@ -144,7 +152,9 @@
       "Amazon.sg Checkout",
       "A preparar o seu pedido",
       "Amazon.com.tr Alışverişi Tamamla",
-      "Kup teraz"
+      "Kup teraz",
+      "Place Your Order - Amazon.co.jp Checkout"
```
Comment (DakkJaniels) [inline config/fairgame.conf:156]: comma missing

File: config/fairgame.conf [hunk 2]
```diff
@@ -163,6 +173,7 @@
       "Obrigado, o seu pedido foi efetuado",
       "Teşekkür Ederiz",
       "Amazon.pl Dziękujemy"
```
Comment (DakkJaniels) [inline config/fairgame.conf:175]: comma missing

**Review by DakkJaniels — CHANGES_REQUESTED** [review]

File: config/fairgame.conf (hunk 1 above)
Comment (delwinbest) [reply config/fairgame.conf]: sorry about that. fixed.

File: config/fairgame.conf (hunk 2 above)
Comment (delwinbest) [reply config/fairgame.conf]: sorry about that. fixed.

**Comment by delwinbest** [comment]
Comma added where missing.

**Review by DakkJaniels — APPROVED** [review]
LGTM thanks!

**Pull request merged**